pkg/host/host_linux.go

```diff
@@ -396,6 +396,7 @@ func init() {
 	checkFeature[FeatureNetworkInjection] = checkNetworkInjection
 	checkFeature[FeatureNetworkDevices] = unconditionallyEnabled
 	checkFeature[FeatureKCSAN] = checkKCSAN
+	checkFeature[FeatureDevlinkPCI] = unconditionallyEnabled
```

Comment:
Please add an actual function that will check presence of the /dev node.
If it's not present we will disable it during fuzzing, in repros, etc.

Reply:
Hmm. There is no related /dev node. However, I can check /sys/bus/pci/devices/0000\:00\:10.0/. What do you think?

Reply:
Okay, I added the check of /sys/bus/pci/devices/0000:00:10.0/